scripts/data/getProtectedAreasWDPAapi.R

```diff
@@ -0,0 +1,105 @@
+library(sf)
+library(rjson)
+library(wdpar)
+library(dplyr)
+remotes::install_github("FRBCesab/worldpa")
+library(worldpa)
+
+# Add inputs
+input <- biab_inputs()
+
+sf::sf_use_s2(FALSE)
+# Get polygon for study area
+# Download the country or state boundary from rnaturalearth
+
+# Read in polygon of study area
+study_area <- input$study_area_polygon
+
+# if input is a geojson, make sure it is in EPSG:4326, otherwise if it is a geopackage, just read in the file 
+study_area <- sf::st_read(study_area) 
+
+# Pull data from wdpa
+countries <- get_countries(key="WDPA_KEY")
```

Comment:
We first need to check whether the key is defined an return an informative error message otherwise.